### DISCUSSION TITLE
Mi Práctica Pastelería (ayuda)

### DISCUSSION THREAD

**Discussion by lunasaurio** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#216] [Q&A]
Que tal compañeros, es la primera vez que tengo un acercamiento al HTML y CSS, y ciertamente estoy muy emocionado por lo que he logrado pero a lo que veo sus páginas están súper desarrolladas y bien hechas, son realmente páginas web, y lo que yo quiero saber es cómo lo logran, es decir esto es a lo máximo que he logrado aspirar (adjunto imágenes) pero me frustra por que a pesar de mis investigaciones y horas de estudio, no logro que realmente parezca una pagina  web, quisiera algo de orientación o algo que me pueda ayudar, si solo es tiempo y práctica, o que puedo hacer para llegar a que parezca un sitio web, bonito. Además de que si entro al código fuente son archivos de CSS realmente grandes, pero como es que sucede, no logro darle formato de página web.

![image](https://user-images.githubusercontent.com/93229275/156616474-399119f6-c022-4661-9681-8afc604d3492.png)
![image](https://user-images.githubusercontent.com/93229275/156616690-5ed23907-270c-4d01-a097-a0202a945428.png)

**Comment by Exusai** [comment]
puedes usar algo como Boostrap, y así puedes importar elementos predefinidos como barras de navegación y cosas así directo a tu html y puedes hacer que tu sitio sea responsivo con poco esfuerzo

**Comment by AngelCruzO** [comment]
No te desanimes, es normal al principio que no te queden bonitas. Tambien no todos tenemos el don del diseño grafico, para mi esta genial tu practica, ya que lo estan intentando por tus medios y no con ayuda de librerias de estilos o plantillas. Despues mejoraras en tus diseños si asi te lo propones.

Animo y muchas felicidades por el logro que haz conseguido

**Reply by lunasaurio** [reply]
Creo que lo mío no es el diseño, gracias por el apoyo

**Reply by AngelCruzO** [reply]
Creeme que yo tampoco hago diseños muy buenos, ni con ayuda de librerias css

**Comment by hackit92** [comment]
Que tal! Lo primero es entender que tu esfuerzo (sin importar que tu resultado sea diferente al de los demás) es lo que más importa, el motivo por el que algunos tenemos páginas más desarrolladas además de un conocimiento previo de estas tecnologías en la mayoría de los casos, es el uso de otras tecnologías como Bootstrap, CSS Flex, JS (entre otras). Si ves mi sitio y abres el código te darás cuenta de que tiene mas de una hoja de estilos, esto es porque una es la que bootstrap incluye y que te permite incluir muchas funciones con esfuerzo mínimo y otra es la que yo hice desde 0. Esto es una práctica muy común, contrario a muchas otras áreas, en la programación no se trata de hacer todo desde 0, se trata de conocer las herramientas y en consecuencia realizar el trabajo mejor y mas rápido. Ojalá nos puedas compartir el enlace de tu página para poder ver tu código y hacer comentarios que te puedan ayudar, quedo a tus órdenes.

**Reply by lunasaurio** [reply]
No lo había visto así, y son herramientas que no conocía para utilizarlas más adelante. Espero poder subirlo pronto para compartirles mi trabajo, muchas gracias.

**Comment by JuanSalvador73** [comment]
Mucho de lo que te ayuda a que tu sitio web quede bien, es experiencia, práctica y dedicación. Lo demás es solo trabajo. Para lograr la parte técnica de estética se maneja mucho en los estilos, esa parte a fondo no la hemos visto aún.

Dado que la práctica requiere solo de momento el esqueleto en HTML, vas mas que bien.

**Reply by lunasaurio** [reply]
Va, muchas gracias

**Comment by carlos-gmx3** [comment]
Pues son ciertas practicas, una común es el uso de un contenedor para las etiquetas del contenido
![imagen](https://user-images.githubusercontent.com/99064239/156684652-f8c2a1e4-53f3-43ea-bb32-212ee69911d7.png)
![imagen](https://user-images.githubusercontent.com/99064239/156684407-8500c360-2664-4a84-8a80-a2d5570cdb2f.png)
![imagen](https://user-images.githubusercontent.com/99064239/156684546-91a86eae-5e6d-4b0c-92cb-6f31ba6adbc3.png)
checa los estilos, header y footer se pueden expandir por todo el ancho disponible y tener sus propios estilos como background-color y demás pero con un div pueden delimitar donde se muestra el contenido y centrarlo
![imagen](https://user-images.githubusercontent.com/99064239/156684775-2d1d4278-e930-461d-8ad9-613e1c7d8728.png)

**Comment by charlyguz** [comment]
Primero que nada no te frustres amigo, todos vamos a diferentes pasos no es una carrera solo tomate tu tiempo también ten en cuenta que muchos ya venían con practica o conocimientos previos, esto solo es cuestión de seguir practicando si quieres mejorar tu diseño en  css te recomiendo este curso en YouTube bro https://www.youtube.com/watch?v=XqFR2lqBYPs
https://www.youtube.com/watch?v=wZniZEbPAzk
también te recomiendo mucho que te apoyes en la comunidad y sobre todo utilices los trabajos que suben nuestros compañeros como un ejemplo y preguntes como lo hicieron mas que una competencia velo como una referencia para hacerlo mejor

**Reply by JuanSalvador73** [reply]
Excelentes recomendaciones, gracias. :D

**Comment by GeraMaldonado** [comment]
Me pasa lo mismo, no es qu eme desanime o me trate de comparar, por curioso vi algunas, y la mía pues no se ve muy bien que digamos, constantemente me digo que es cuestión de estudiar, practicar, que los resultados no se ven tan rápido, pero honestamente si me dio algo de pena y siento que me detuvo un poco

**Reply by carlos-gmx3** [reply]
Entender que hacen los elementos de forma individual es muuuy importante, después te será mas fácil jugar con los elementos.
Igual podrías preguntar aquí o en google como hacer ciertas cosas, apenas busqué en internet con hacer un dropdown menú con html y css

**Reply by villegas97** [reply]
Amigo, no tengas pena, son personas que quieren presumir sus conocimientos, la verdad a mi no me parece lo más correcto que se avienten páginas con bootstrap muy bien diseñadas precisamente para que no pase esto, que los que apenas van aprendiendo se desanimen, yo llevo unos meses trabajando como desarrollador web y aún así no me he enfocado tanto al diseño, pues lo que mi trabajo requiere más es backend y te puedo decir que yo pensaba mandar puro HTML, puesto que es el tema al que corresponde la práctica, no bootstrap ni CSS.

**Reply by hackit92** [reply]
@GeraMaldonado no te preocupes por el resultado, estas prácticas solo se tratan de que aprendas como se utiliza HTML y si la curiosidad y el tiempo te dan, te adentres en otras tecnologías (lo cual es el día a día del programador) para las personas que ya tenemos más experiencia es relativamente fácil construir una landing vistosa, sin embargo eso no significa que no haya muchísimas horas de estudio detrás, por lo que compararte sería injusto cuando no podemos medir cuantas horas nos llevan algunas personas. Por otro lado, veo que te comentan que no es correcto hacer uso de otras tecnologías porque no es hacer desde 0 y es una visión errónea de lo que estamos haciendo, la curiosidad por nuevas herramientas te hace un profesional más completo, en el día a día no sirve tener a alguien como loco 16 horas armando una hoja de estilos, cuando existen frameworks excelentes que si conoces puedes usar y personalizar a tus necesidades. Te recomiendo darte una vuelta por faztweb.com, hay buenos tutoriales sobre varias tecnologías y por favor no caigas en desanimo, launchx es una comunidad y estoy seguro que nadie hará de menos tu trabajo aun cuando lo consideres sencillo, sigue esforzandote y en las próximas semanas verás el fruto de tu esfuerzo, saludos!

**Comment by LuceroLuciano** [comment]
Es sorprendente el esfuerzo  que haces y quiero felicitarte por ello, realmente has hecho un gran trabajo. Tu pagina quedo increíble! 
Creo que con la practica constante y el tiempo harás cientos de miles de paginas muy cool. 
Yo tampoco soy tan pro como lxs demas explorer que la están rompiendo con sus paginas bien bonitas, pero me encanta saber que hay explores dispuestos ayudar y que no nos vamos a rendir fácilmente. 
Así que como dice el MC "De aquí hasta el espacio y mas allá."

**Comment by MarcosSilvaG** [comment]
Antes que nada tranquilo, no te agobies porque no te quedan tan bien como quisieras, todo es cuestion de tiempo, práctica y estudio.
Las primeras páginas siempre son un poco rusticas pero para iniciar no está tan mal, ya que estamos en un proceso de aprendizaje :D y mas bien miralo como eso, aprendizaje, personalmente el diseño se me da particularmente mal por lo que he estado haciedo para prácticar ese punto es "copiar" la maquetación de páginas web ya existentes y con copiar me refiero a sacar una captura de la web e intentar replicarla con código, aún me lleva algo de tiempo pero cada vez siento que mejoro un poco o que ya tengo mas ideas de como maquetarla. espero te sirva :D  y no te rindas (y)

**Comment by villegas97** [comment]
Amigo, darle ese diseño profesional con puro HTML y CSS puede tomar demasiado tiempo y te puedo asegurar que esas páginas bien elaboradas que ves no están hechas a puro código desde 0, utilizan bootstrap, he visto una que otra y al inspeccionar elemento y tan solo ver el nombre de las clases que utilizan te das cuenta, para serte sincero también lo usaré para hacer algo rápido puesto que no he tenido ni cuento ya con tanto tiempo para hacerla (hoy se entrega) pero no te desanimes, si no estuvieras haciendo mucho esfuerzo no te desanimarías al ver lo de los demás y eso es lo que cuenta, porque si sigues con esa dedicación y con una buena ruta de aprendizaje lograrás grandes cosas, animo. Te dejo un link para que aprendas CSS de una forma "rapida" y sencilla: https://www.w3schools.com/css/  son tutoriales básicos pero que en conjunto son necesarios, recuerda que todo, por más grande que sea, se compone de partes pequeñas, en su tiempo me hice los tutoriales de HTML de esa página y la verdad que fue de gran ayuda.

PD: El tema al que corresponde la práctica es HTML, por ende tu trabajo está bien, ya hiciste hasta más porque le agregaste CSS.

**Comment by jesusbeb** [comment]
Amigo, tu tranquilo!!! Si te das cuenta el mission comander siempre ha dicho que no nos estresemos jeje. Apenas vamos empezando con codigo HTML, por lo que la página aun no necesita tener un gran diseño, eso se hara con CSS y esta practica corresponde solo a HTML, vamos por pasos. Quienes ya han hecho paginas muy vistosas es porque ya tienen conocimiento previo, pero si partimos de que aun no llegamos a temas de diseño con CSS, entonces basta con que tu pagina sea funcional, recuerda que es el puro esqueleto. Saludos

**Comment by ArmandoRiveraT** [comment]
No te presiones, por ahora lo importante es saber hacer la maquetación html, ya lo de meterle css y demás cosas solo es un bonus como lo dice en la actividad.

**Comment by ChristianDeM** [comment]
Como dicen no te desanimes  yo ni por que uso bootsrap me gustan mis diseños.  siento que es por falta  de conceptos de diseño grafico como tal.  horita que andas empezando es normal que te sientas un poco perdido, pero cada hora que le metas a entender como es maquetar se  te sera mas sencillo.  :)
te recomiendo que aprendas bien como  usar  los grids ayuda bastante al mentalizar tus ideas.
ya teniendo el conepto de maquetacion dale un vistaso a bootsrap  se te ara super sencillo de entender  y sobre todo crear una web super rapida. ya con el tiempo puedes  usar mas cosas como fontawesome  y animated.css    cuando menos te los esperar boom paginas web en menos de unas horas.  
Suerte

**Comment by lunasaurio** [comment]
Que tal compañeros muchas gracias a todos por sus respuestas, he terminado el código y no encuentro como subirlo, si alguien sabe apreciaría su ayuda
